Production/govuk_ios/Helpers/StringValidator/StringValidator.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+struct StringValidator {
```

Comment:
Should this be a `ValidatorProvider`?

Reply:
currently the protocol is called ValidatorProvider as we agreed it should stay

Reply:
Yeah, should this not conform to that protocol? As it is a validator

Reply:
doen